.rubocop.yml

```diff
@@ -14,20 +14,16 @@ Style/StringLiterals:
 Style/ClassCheck:
   EnforcedStyle: kind_of?
 
-# It's better to be more explicit about the type
-Style/BracesAroundHashParameters:
-  Enabled: false
-
```

Comment:
The cop in question has been removed below:
- https://github.com/rubocop/rubocop/issues/7641